src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx

```diff
@@ -174,18 +174,14 @@ Our .NET agent relies on environment variables to tell the .NET Common Language
     ```
```

Comment:
Perhaps update line 172 to say `**Windows (using MSI Installer):**` 

I can't add a comment on that line since it's outside of the context of the PR